packages/angular/stories/pie-donut.stories.ts

```diff
@@ -42,7 +45,7 @@ donutStories.add(donutOptions.title, () => ({
 		imports: [ChartsModule]
 	},
 	props: {
-		data: donutData,
-		options: addWidthAndHeight(donutOptions)
+		data: object("Data", donutData),
+		options: object("Options", addWidthAndHeight(donutOptions))
 	}
 }));
```

Comment:
Donut is missing `donutStories.addDecorator(withKnobs);`

The donut knobs don't update because of this:

![image](https://user-images.githubusercontent.com/14989804/71192626-a6408700-2256-11ea-9d11-e28048c7b2fa.png)